js/common/test/unit-tests/tensor/constructor-type.ts

```diff
@@ -82,6 +82,14 @@ describe('Tensor Constructor Tests - check types', () => {
     assert.equal(tensor.type, 'bool', "tensor.type should be 'bool'");
   });
 
+  it('[uint8] new Tensor(uint8ClampedArray, dims): uint8 tensor can be constructed from Uint8ClampedArray', () => {
+    const uint8ClampedArray = new Uint8ClampedArray(2);
+    uint8ClampedArray[0] = 0;
+    uint8ClampedArray[1] = 256; // clamped
+    const tensor = new Tensor('uint8', uint8ClampedArray, [2]);
```

Comment:
this will fail the build because also need to add `new Tensor('uint8', Uint8ClampedArray, dims)` in lib/tensor.ts